ci/builders/windows_arm_host_engine.json

```diff
@@ -0,0 +1,124 @@
+                        "out/host_debug_arm64/zip_archives/windows-arm64/flutter-cpp-client-wrapper.zip",
+                        "out/host_debug_arm64/zip_archives/flutter-web-sdk-windows-arm64.zip"
+                    ],
+                    "name": "host_debug_arm64"
+                }
+            ],
+            "drone_dimensions": [
+                "device_type=none",
+                "os=Windows-10"
+            ],
+            "gclient_custom_vars": {
+                "download_android_deps": false
+            },
+            "gn": [
+                "--runtime-mode",
+                "debug",
+                "--full-dart-sdk",
+                "--no-lto",
+                "--windows-cpu",
+                "arm64"
+            ],
+            "name": "host_debug_arm64",
+            "ninja": {
+                "config": "host_debug_arm64",
+                "targets": [
+                    "flutter/build/archives:artifacts",
+                    "flutter/build/archives:embedder",
+                    "flutter/tools/font-subset",
+                    "flutter/build/archives:dart_sdk_archive",
+                    "flutter/shell/platform/windows/client_wrapper:client_wrapper_archive",
+                    "flutter/build/archives:windows_flutter",
+                    "flutter/web_sdk"
+                ]
+            },
+            "tests": []
+        },
+        {
+            "archives": [
+                {
+                    "base_path": "out/host_profile_arm64/zip_archives/",
+                    "type": "gcs",
+                    "include_paths": [
+                        "out/host_profile_arm64/zip_archives/windows-arm64-profile/windows-arm64-flutter.zip"
+                    ],
+                    "name": "host_profile_arm64"
+                }
+            ],
+            "drone_dimensions": [
+                "device_type=none",
+                "os=Windows-10"
+            ],
+            "gclient_custom_vars": {
+                "download_android_deps": false
+            },
+            "gn": [
+                "--runtime-mode",
+                "profile",
+                "--no-lto",
+                "--windows-cpu",
+                "arm64"
+            ],
+            "name": "host_profile_arm64",
+            "ninja": {
+                "config": "host_profile_arm64",
+                "targets": [
+                    "windows",
+                    "gen_snapshot",
+                    "flutter/build/archives:windows_flutter"
+                ]
+            },
+            "tests": []
+        },
+        {
+            "archives": [
+                {
+                    "base_path": "out/host_release_arm64/zip_archives/",
+                    "type": "gcs",
+                    "include_paths": [
+                        "out/host_release_arm64/zip_archives/windows-arm64-release/windows-arm64-flutter.zip"
```

Comment:
When I tested this on my x64 machine I got this archive instead: `out/host_release_arm64/zip_archives/windows-x64-release/windows-x64-flutter.zip`

I suspect the output path needs to be updated to use the `$full_target_platform_name` instead: https://github.com/flutter/engine/blob/6e6a6538f9e217b0a5fb94550cf16d7c12ec2c3d/build/archives/BUILD.gn#L227-L230


Here are the commands I used to build:

```
.\tools\gn --runtime-mode release --no-lto --no-goma --windows-cpu arm64
autoninja -C ..\out\host_release_arm64 windows gen_snapshot flutter/build/archives:windows_flutter
```

Reply:
I missed that one (windows-arm64 vs windows-arm64-{debug,release}), it's now fixed.
Not sure if target_cpu is always defined for x64 build, but we'll see that with CI.